src/Libraries/CoreNodeModels/ColorPalette.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Xml;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Media;
+using ProtoCore.AST.AssociativeAST;
+using DSColor = DSCore.Color;
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph;
+
+using CoreNodeModels.Properties;
+
+namespace ColorNode
+{
+    /// <summary>
+    ///     Custom Color UI node.
+    /// </summary>
+    [NodeName("Color Picker")]
+    [NodeDescription("ColorPaletteDescription", typeof(Resources))]
+    [NodeCategory("Core.Color.Create")]
+    [NodeSearchTags("ColorPaletteSearchTags", typeof(Resources))]
+    [IsDesignScriptCompatible]
+    [OutPortNames("Color")]
+    [OutPortTypes("Color")]
+    [OutPortDescriptions("Selected Color.")]
+    public class nColorPicker : NodeModel, INotifyPropertyChanged
+    {
+        private Color scolor = Color.FromArgb(255, 0, 0, 0);
+        private ObservableCollection<Xceed.Wpf.Toolkit.ColorItem> cList = new ObservableCollection<Xceed.Wpf.Toolkit.ColorItem>()
+        {
```

Comment:
@Gytaco is this file here in error - it looks like you did not touch this file at all in your update?